### DISCUSSION TITLE
mac m1 make fail

### DISCUSSION THREAD

**Discussion by laoshuaichiyu** [ImageMagick/ImageMagick#7251] [Help] [answered]
### ImageMagick version

7.1.1-30

### Operating system, version and so on

mac m1 macos 14.4

### Description

this my confgure
./configure --disable-openmp

 jpeg-xl is available in pkg-config:  
pkg-config --list-all
...
 libjxl                   libjxl - Loads and saves JPEG XL files
...

make result
coders/jxl.c:64:10: fatal error: 'jxl/decode.h' file not found
#include <jxl/decode.h>
         ^~~~~~~~~~~~~~
1 error generated.
gmake[1]: *** [Makefile:10361: coders/MagickCore_libMagickCore_7_Q16HDRI_la-jxl.lo] Error 1

**Comment by dlemstra** [comment, marked as answer]
Your question was already answered here: #7111

**Reply by laoshuaichiyu** [reply]
Thank you. I only searched in 'Discussions' and forgot' Issues'.
